src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.html

```diff
@@ -28,7 +28,7 @@
           <!-- Icon for the expandable -->
           <div class="container" *ngIf="interactor.isSubComplex">
             <a title="{{ interactor.interactor.name }} interactors" (click)="toggleSubcomplexExpandable(i)">
```

Comment:
`<a title="{{ interactor.interactor.name }} interactors"...` should be `<a title="{{ interactor.interactor.name }} components" ...`

Reply:
Same in lines 58 and 61